crewai_tools/tools/stagehand_tool/stagehand_tool.py

```diff
@@ -323,13 +345,17 @@ def close(self):
                 model_name=self.model_name,
                 self_heal=self.self_heal,
                 wait_for_captcha_solves=self.wait_for_captcha_solves,
-                model_client_options=model_client_options,
+                model_api_key=self.model_api_key,
```

Comment:
I might be wrong but I think the config does not support this attribute. It should be provided directly on Stagehand